client/signup/jetpack-connect/sso.jsx

```diff
@@ -340,8 +341,8 @@ const JetpackSSOForm = React.createClass( {
 				buttons={ buttons }
 				onClose={ this.closeTermsDialog }
 				isVisible={ this.state.showTermsDialog }
-				className="jetpack-connect_sso_terms-dialog">
-				<div className="jetpack-connect_sso_terms-dialog-content">
+				className="jetpack-connect__sso_terms-dialog">
```

Comment:
After sso there was sometimes a single underscore, and sometimes a double underscore.
Addressed it here: https://github.com/Automattic/wp-calypso/pull/6986/commits/52cd23647f6c90f4212a950cf3b89dbd2e7e3880